src/commands/init.js

```diff
@@ -59,22 +64,32 @@ export async function init(flags) {
 			commands: data.cmds_dir,
 		},
 	};
+
 	const file = JSON.stringify(config, null, 2);
 
-	if (Object.keys(data).length < 4) process.exit(1);
+	if (
+		(Object.keys(data).length < 4 && !flags.sync) ||
```

Comment:
lift this condition into a boolean variable so its more readable as to what this is doing

Reply:
like this?
```js
const requiredData = flags.sync ? 3 : 4;
if (Object.keys(data).length < requiredData)
```

Reply:
lift the whole expression including Object.keys(...) into one variable. I find it more readable if we give names to conditions

Reply:
```js
const requiredData = flags.sync ? 3 : 4;
const receivedData = Object.keys(data).length;
if (receivedData < requiredData)
```
👀

Reply:
const hasRequiredData = flags.sync && Object.keys(data)

Reply:
```javascript
const hasSyncFlag = flags.sync !== undefined
const lessThan4Keys = Object,keys(data).length < 4

if((!hasSyncFlag && lessThan4Keys) || (hasSyncFlag && !lessThan4Keys)) {

}
```
maybe something like this would work?